src/server/db/mongo/user-privat-provider.js

```diff
@@ -1,13 +1,42 @@
 //@flow
+import { sha3 } from 'web3-utils'
 import UserPrivateModel from './models/user-private.js'
+import PropsModel from './models/props.js'
+import { get } from 'lodash'
 import logger from '../../../imports/logger'
 import { type UserRecord } from '../../../imports/types'
 
 class UserPrivate {
   constructor() {
     this.model = UserPrivateModel
+    this.upgrade()
+      .then(_ => logger.info('upgrade done'))
+      .catch(e => logger.error('upgrade failed', { err: e.message, e }))
   }
 
+  async upgrade() {
```

Comment:
Generally the things like those one shouldn't be part of the application
And of course shouldn't try to be applied on app's startup.
It's something like 'fixture' or 'seed' and generally contains in some cli scripts could be ran while app deploys/upgrades.

I propose to add simple cli console to out app built over ready-to-use solutions such as 
https://www.npmjs.com/package/yargs
https://github.com/oclif/oclif

Then, create command class (or callback function - depending or library you'll choose) and move this code inside it
Finally, include this command call to the deploy scripts

Reply:
done